### PR TITLE
Exporting SSSOM mappings

### PR DESCRIPTION
**Pull request by matentzn**
### Outstanding action items:

- [x] Update make sure OMIM mappings include OMIMPS terms!
- [x] Make sure that ICD10 mappings are also exported (only ICD10CM prefix).
- [x] Add remaining metadata:
   - [x] doid:
      - [x] object_source_version: http://purl.obolibrary.org/obo/doid/releases/2018-07-05/doid.owl
      - [x] mapping_date: 2018-07-05
   - [x] ncit: 
      - [x] mapping_date: 2018-07-05
    - [x] ordo: 
       - [x] mapping_date: 2021-07-06
       - [x] object_source_version: http://www.orphadata.org/data/ORDO/ordo_orphanet_3.1.owl
    - [x] omim:
       - [x] mapping_date: 2022-02-11
       - [x] object_source_version: http://purl.obolibrary.org/obo/mondo/sources/2022-02-11/omim.owl
    - [x] everything else (every other source) should be `mapping_date: 2017-09-20`

### CODE REVIEW

File: src/ontology/mappings/mondo_broadmatch_ncit.sssom.tsv [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#curie_map:
```
Comment (cmungall) [inline src/ontology/mappings/mondo_broadmatch_ncit.sssom.tsv:1]: let's have a policy of only equivs for ncit

**Review by cmungall — COMMENTED** [review]
can we include more

 - more info in headers
    - separate date when mappings were made vs exported
    - versions of ontologies
 - more info in rows
    - labels
    - (later) some boomer stats

File: src/ontology/mappings/mondo_broadmatch_ncit.sssom.tsv (hunk 1 above)
Comment (matentzn) [reply src/ontology/mappings/mondo_broadmatch_ncit.sssom.tsv:1]: Made a ticket here

Comment (matentzn) [reply src/ontology/mappings/mondo_broadmatch_ncit.sssom.tsv:1]: https://github.com/monarch-initiative/mondo/issues/3097

**Review by hrshdhgd — COMMENTED** [review]
Looks good to me. Are there tests for this?

**Comment by joeflack4** [comment]
@matentzn 49 files changed! Are you just wanting me to review the `.py` files mainly?

**Comment by hrshdhgd** [comment]
Hey @joeflack4 , I'll let Nico answer but 47 of the 49 are sssom TSVs generated. Just thought of letting you know.

**Comment by nicolevasilevsky** [comment]
note to self, I don't need to review this

**Comment by matentzn** [comment]
- doid:
   - object_source_version: http://purl.obolibrary.org/obo/doid/releases/2018-07-05/doid.owl
   - mapping_date: 2018-07-05
- ncit: 
    - mapping_date: 2018-07-05
- ordo: 
    - mapping_date: 2021-07-06
    - object_source_version: http://www.orphadata.org/data/ORDO/ordo_orphanet_3.1.owl
- omim:
   - mapping_date: 2022-02-11
   - object_source_version: http://purl.obolibrary.org/obo/mondo/sources/2022-02-11/omim.owl

**Comment by matentzn** [comment]
Update make sure OMIM mappings include OMIMPS terms!

**Comment by hrshdhgd** [comment]
restoring this branch again. Have addressed https://github.com/mapping-commons/sssom-py/issues/214 and fix applied to this branch for new mappings.

**Pull request merged**